### PR TITLE
Version 6 migration guide

### PR DESCRIPTION
**Pull request by jkoenig134**

### CODE REVIEW

**Review by tnotheis — APPROVED** [review]

File: _docs_integrate/migration-from-v5-to-v6.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - component: integrate
+  - level:
+  - implementation status: DOCS ONLY
+  - documentation status: DONE
+  - published: true
+  - link: migration-from-v4-to-v5
+require:
+required_by:
+# End automatic generation
+---
```
[the remark this answers is not in the document]
Comment (jkoenig134) [reply _docs_integrate/migration-from-v5-to-v6.md:22]: b/c I hate that tool and will not touch it :)

File: _docs_integrate/data-model-overview.md
[the remark this answers is not in the document]
Comment (jkoenig134) [reply _docs_integrate/data-model-overview.md:1]: on occurence is valid

File: _docs_integrate/establish-relationships.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ If the RelationshipTemplate is intended to establish a Relationship between the
 
 If you have successfully created the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) on the templator, you will receive a success response from which you can read its `id`. As the templator is the creator of the RelationshipTemplate, the `createdBy` property contains the Address of the templator. For this reason, the value of the `isOwn` property is set to `true` in this context.
 
-{% include copy-notice description="Save the `id` of the RelationshipTemplate so that you can refer to it and make it available to other Identities later. For the same reason, save the values of the properties `truncatedReference` and `secretKey`." %}
+{% include copy-notice description="Save the `id` of the RelationshipTemplate so that you can refer to it and make it available to other Identities later. For the same reason, save the values of the property `truncatedReference`." %}
```
[the remark this answers is not in the document]
Comment (jkoenig134) [reply _docs_integrate/establish-relationships.md:82]: I think you mean of? changed.

File: _docs_integrate/migration-from-v5-to-v6.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply _docs_integrate/migration-from-v5-to-v6.md:22]: https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExdHY5MjMwM21oemQ3N3hoajk5aXZnNXJsZXZ3cDkxaGxtdmZqeTlnZyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/s4T64Kx2p0fhm/giphy.gif

**Pull request merged**